.github/workflows/windows.yml

```diff
@@ -124,7 +124,8 @@ jobs:
 
       # Label as 'windows-build-test' for testing purposes. To use, remove 'false &&' from the 'if' condition and change label to 'unstable' after testing
```

Comment:
Is it worth updating this comment now that the approach has changed?

Also, I was just wondering why we're using an `if` statement to stop this running rather than commenting it out in the workflow file? It seems a shame to have to evaluate the logic each time, is there an advantage to leaving it in that I'm not considering?

Reply:
Good spot, I have a bad habit of not spotting when comments need updating.

I'm not a fan of commented out code. Gives a higher risk of it not working when uncommented later. Actually now thinking that the commented `if ...` should be removed, and the advice should be to copy it back from the conda work flow if needed. That way it wont get out of date with any changes of when to build.